eservice/lib/libpdo_enclave/contract_enclave.cpp

```diff
@@ -99,6 +99,38 @@ pdo_err_t ecall_ShutdownContractWorker(){
     return result;
 }
 
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+pdo_err_t ecall_CalculateSealedContractKeySize(
+  size_t contractIdSize,
+  size_t* pSealedContractKeySize)
+{
```

Comment:
for ecall parameters... could you please use the in/out convention?

Reply:
should be fixed now